Dockerfile

```diff
@@ -0,0 +1,40 @@
+# See https://testdriven.io/blog/docker-best-practices/ for recommendations
```

Comment:
Do we really need to provide/maintain a dockerfile for this simple, pure python library?

Reply:
probably not! I think only for the vodf-schema repo. I'll remove this and clean up the CI